hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1021,6 +1021,49 @@ function upgrade_1023() {
     return true;
   }
 
+  /**
+   * Creates Option Group and Option Values for Health an Life Insurance Plan 
+   * Types.
+   * 
+   * @return boolean TRUE
+   */
+  public function upgrade_1025() {
```

Comment:
I tried using upgrade_1024, but for some reason it would not get identified as a pending upgrade with this method name, so I just bumped it to 1025, where it started to work.  I saw Davi ran into a similar problem, but couldn't understand why it was happening.

https://github.com/civicrm/civihr/commit/cbe0b5b6c0a6caca78700387b500f18cd5a4cd5f